gulp/tasks/test_ci.js

```diff
@@ -0,0 +1,14 @@
+(function () {
```

Comment:
@peter-fogg this looks like the same file as `gulp/tasks/test.js` with the exception of `'test_ci'` vs. `'test'` on line 9. Is there a way that we can implement local vs CI testing as the same `'test'` task, but parameterize that task's set of browsers, or more generally, its config file?